scripts/unicode.py

```diff
@@ -431,3 +431,22 @@ def emit_break_module(f, break_table, break_cats, name):
             sentence_table.extend([(x, y, cat) for (x, y) in sentence_cats[cat]])
         sentence_table.sort(key=lambda w: w[0])
         emit_break_module(rf, sentence_table, list(sentence_cats.keys()), "sentence")
+
+        rf.write("""
```

Comment:
nit: no, let's not have too much code generation like this. they can live in a different module somewhere